### ISSUE TITLE
app.initialized() returns a promise that never gets resolved

### ISSUE THREAD

**Issue by rade363** [freshworks-developers/react_todo_app#1]
Hello!

I am facing an issue when I test this sample app locally. 

For some reason the `app.initialized()` promise never gets resolved. This makes the `child` component always to stay as `<h3>App is loading</h3>`.
The global `app` variable contains the `initialized` method, so this means that the Freshdesk client script gets loaded successfully. 
However, the `.then()` callback never gets executed, there are no errors in `.catch()`, and even nothing in `.finally()`. 

My specs:
- fdk 6.10.0
- node 10.18.1
- Windows 10x64
- Chrome 90.0.4430.212 (Stable), AdBlockers disabled

**Comment by alan-cooney** [comment]
Also have this. For now I've just loaded directly in index.html

```
<script src="https://static.freshdev.io/fdk/2.0/assets/fresh_client.js"></script>
```

**Comment by rade363** [comment]
@alan-cooney yep, this is the workaround I had to use, too. Here is a [thread](https://community.developers.freshworks.com/t/freshdesk-client-app-initialized-returns-a-promise-that-never-gets-resolved/2871) regarding this issue on Dev Community.
